src/Mvc/Mvc.Core/src/ApplicationParts/IAssemblyPartResolver.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Mvc.ApplicationParts
+{
+    public interface IAssemblyPartResolver
```

Comment:
How does one register an instance of this?

Reply:
This shouldn't be in the commits anymore. It should be in Mvc.Razor.RuntimeCompilation/src i suppose? Also i would imagine this should be registerable through the runtime compilation options.

Reply:
i've removed this file from the PR since it was a duplicated file